src/Microsoft.AspNetCore.Mvc.TagHelpers/EnvironmentTagHelper.cs

```diff
@@ -11,10 +11,14 @@ namespace Microsoft.AspNetCore.Mvc.TagHelpers
     /// <summary>
     /// <see cref="ITagHelper"/> implementation targeting &lt;environment&gt; elements that conditionally renders
     /// content based on the current value of <see cref="IHostingEnvironment.EnvironmentName"/>.
+    /// If the environment is not listed in explicitly identified <see cref="Names"/> or <see cref="Include"/> lists or
+    /// if it is in the <see cref="Exclude"/> list, the content will not be rendered. 
     /// </summary>
     public class EnvironmentTagHelper : TagHelper
     {
         private static readonly char[] NameSeparator = new[] { ',' };
+        private static readonly char[] IncludeEnvironmentsSeparator = new[] { ',' };
+        private static readonly char[] ExcludeEnvironmentsSeparator = new[] { ',' };
```

Comment:
Why imply we'd ever use different separators for `Include` and `Exclude`? Can use `NameSeparator` everywhere.